atom.mk

```diff
@@ -117,5 +119,10 @@ LOCAL_LIBRARIES := cryptsetup
 
 LOCAL_COPY_FILES := scripts/volmount/dm:lib/pv/volmount/dm
 
-include $(BUILD_CUSTOM)
```

Comment:
you broke the other module here... keep the BUILD_CUSTOM please to end each module...